### PR TITLE
Fix for CI repo and branch

### PR DESCRIPTION
**Pull request by foreseaz**
Quick fix for CI repo and branch, which blocks #17

### CODE REVIEW

File: ci-scripts/run-integration-tests.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@
 set -e
 
 # Build and start chain Devnet
-git clone https://github.com/calvinlauco/chain || echo "`chain` folder already exists"
-cd chain; git checkout feature/cro-629-travis-integration-tests; git pull; cd ..;
+git clone https://github.com/crypto-com/chain || echo "`chain` folder already exists"
+cd chain; git checkout master; git pull; cd ..;
```
Comment (calvinlauco) [inline ci-scripts/run-integration-tests.sh]: The old way of running integration tests is deprecated in `chain` repository. While migration to the new drone.io would take some time (and is in backlog), consider pointing the integration tests to:

https://github.com/crypto-com/chain/tree/438b62c07a3d91760d1bebabc583af8c9242ca16/integration-tests

at the moment.

Comment (foreseaz) [reply ci-scripts/run-integration-tests.sh]: Sure thing~

**Comment by foreseaz** [comment]
@calvinlauco ,  `git checkout 438b62c07a3d91760d1bebabc583af8c9242ca16 -b integratin-test` should be okay?

**Comment by calvinaco** [comment]
Closing in favour of https://github.com/crypto-com/cro-nodelib/pull/69

**Pull request closed without merging**